account-kit/privy-integration/src/types.ts

```diff
@@ -0,0 +1,177 @@
+import type { Address, Hash, Hex } from "viem";
+import type { swapActions } from "@account-kit/wallet-client/experimental";
+
+/**
+ * Configuration for the Alchemy provider
+ */
+export interface AlchemyProviderConfig {
+  /** Alchemy API key for @account-kit/infra transport */
+  apiKey?: string;
+
+  /** JWT token for authentication */
+  jwt?: string;
+
+  /** Custom RPC URL */
+  url?: string;
+
+  /** Policy ID(s) for gas sponsorship */
+  policyId?: string | string[];
+
+  /**
+   * Default: true → try to sponsor via Alchemy Gas Manager
+   * Set to false to disable sponsorship by default
+   */
+  defaultSponsored?: boolean;
+}
+
+/**
+ * Unsigned transaction request
+ */
+export interface UnsignedTransactionRequest {
+  /** Recipient address */
+  to: Address;
+
+  /** Transaction data (calldata) */
+  data?: Hex;
+
+  /** Transaction value - accepts string | number | bigint */
+  value?: string | number | bigint;
+}
+
+/**
+ * Options for sending a transaction
+ */
+export interface SendTransactionOptions {
+  /**
+   * Whether to sponsor the transaction
+   * Default: true if policy ID exists and defaultSponsored is not set to false
+   */
+  sponsored?: boolean;
```

Comment:
same here, if it doesn't go against existing patterns, would prefer default false booleans like `skipSponsorship`.